docs/static/monitoring/monitoring-internal-legacy.asciidoc

```diff
@@ -0,0 +1,152 @@
+[role="xpack"]
+[[monitoring-internal-collection-legacy]]
+=== Collect {ls} monitoring data using legacy internal collectors
```

Comment:
Do we have a description of what is meant by 'legacy'?

Reply:
We have notes at key entry points directing users to one of the other options. What are you recommending? 
 
<img width="828" alt="Screen Shot 2020-04-09 at 3 56 14 PM" src="https://user-images.githubusercontent.com/35154725/78935396-ade0a180-7a7a-11ea-9ef4-26a55d19166b.png">